jobs/service-fabrik-serviceflow-manager/templates/bin/service-fabrik-serviceflow-manager_ctl.erb

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Will be a good idea to use bpm based jobs going forward. You can explore that.
I'm not very familiar with intricacies but @subhankarc has written `service-fabric-apiserver` bpm based job

Reply:
Done!